stairway/src/test/java/bio/terra/stairway/impl/EnumerateFlightsTest.java

```diff
@@ -146,14 +148,26 @@ public void enumTest() throws Exception {
     flightEnum = flightDao.getFlights(pageTokenString, 3, filter);
     checkResults("case 10", flightEnum.getFlightStateList(), Arrays.asList("3", "4", "5"));
     assertThat(flightEnum.getTotalFlights(), equalTo(6));
+
+    // Case 11: sorting in ascending order
+    filter =
+            new FlightFilter()
+                    .submittedTimeSortDirection(FlightFilterSortDirection.ASC);
+    flightList = flightDao.getFlights(0, 100, filter);
+    checkResults("case 11", flightList, Arrays.asList("0", "1", "2", "3", "4", "5"));
+
+    // Case 12: sorting in descending order
+    filter =
+            new FlightFilter()
+                    .submittedTimeSortDirection(FlightFilterSortDirection.DESC);
+    flightList = flightDao.getFlights(0, 100, filter);
+    checkResults("case 12", flightList, Arrays.asList("5", "4", "3", "2", "1", "0"));
   }
 
   private void checkResults(String name, List<FlightState> resultlList, List<String> expectedIds) {
-    assertThat(
-        name + ": right number of elements", resultlList.size(), equalTo(expectedIds.size()));
     List<String> actualIds =
         resultlList.stream().map(FlightState::getFlightId).collect(Collectors.toList());
-    assertTrue(CollectionUtils.isEqualCollection(actualIds, expectedIds), "elements match");
+    assertThat(name + ": elements match in the correct order", actualIds, contains(expectedIds.toArray()));
```

Comment:
Is `.toArray()` needed here?
```suggestion
    assertThat(name + ": elements match in the correct order", actualIds, contains(expectedIds));
```

Reply:
It is...since contains takes a vararg.  If you leave it as a collection, it will treat it as a single item in the list that happens to be a collection (I've been bitten by that many times)